src/webgpu/api/validation/buffer/mapping.spec.ts

```diff
@@ -12,6 +12,7 @@ import { ValidationTest } from '../validation_test.js';
 class F extends ValidationTest {
   async testMapAsyncCall(
     success: boolean,
+    validationError: boolean,
     earlyRejection: boolean,
     rejectName: string | null,
```

Comment:
This is becoming a bit of a "boolean soup" where it's hard to see what the arguments are at the call sites.
Can we replace it with something a bit richer? Example:

```suggestion
    expectation: 'success' | { validationError: boolean, earlyRejection: boolean, rejectName: string },
```

Reply:
Thanks for the suggestion. Good idea. Updated.